src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2171,7 +2171,7 @@ def config_slot_auto_swap(cmd, resource_group_name, webapp, slot, auto_swap_slot
     client = web_client_factory(cmd.cli_ctx)
     site_config = client.web_apps.get_configuration_slot(resource_group_name, webapp, slot)
     site_config.auto_swap_slot_name = '' if disable else (auto_swap_slot or 'production')
-    return client.web_apps.update_configuration_slot(resource_group_name, webapp, site_config, slot)
+    return client.web_apps.update_configuration_slot(resource_group_name, webapp, slot, site_config)
```

Comment:
Same comment here as well plus make sure to use use positional arguments here, so this doesn't cause issues in the future.